dist/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="/style.e308ff8e.css">
+  <script src="/style.e308ff8e.js"></script></head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()" method="post">
+      <fieldset class="input_area" name="personal-information" id="personal-information">
+        <legend>Personal information</legend>
+
+        <div class="surname input_field">
+          <label for="surname">Surname:</label>
+          <input type="text" id="surname" name="surname" required="" autocomplete="off">
+        </div>
+
+        <div class="name input_field">
+          <label for="name">Name:</label>
+          <input type="text" id="name" name="name" required="" autocomplete="off">
```

Comment:
fix everywhere, just 'required'

```suggestion
          <input type="text" id="name" name="name" required autocomplete="off">
```

Reply:
done